presto-main/src/test/java/com/facebook/presto/sql/planner/TestDomainTranslator.java

```diff
@@ -909,6 +909,14 @@ public void testFromInPredicateWithCastsAndNulls()
                 new InListExpression(ImmutableList.of(cast(toExpression(null, SMALLINT), BIGINT), toExpression(1L, SMALLINT))))));
     }
 
+    @Test
+    public void testFromInPredicateWithFunctions()
```

Comment:
I would like you to add (or make sure they already exists) for: 
 - `x in (1,2,3,y)`
 - `x not in (1,2,3,y)`
 - `y = 4 and x in (1,2,3,y)`
 - `y = 4 and x not in (1,2,3,y)`